src/index.tsx

```diff
@@ -13,11 +13,13 @@ const root = ReactDOMClient.createRoot(container)
 const LazyConfig = React.lazy(async () => import('./pages/config.jsx'))
 const LazyHelperUi = React.lazy(async () => import('./pages/helper-ui.jsx'))
 const LazyRedirectPage = React.lazy(async () => import('./pages/redirect-page.jsx'))
+const LazyInterstitial = React.lazy(async () => import('./pages/redirects-interstitial.jsx'))
 
 const routes: Route[] = [
   { default: true, component: LazyHelperUi },
-  { path: '#/ipfs-sw-config', shouldRender: async () => (await import('./lib/routing-render-checks')).shouldRenderConfigPage(), component: LazyConfig },
-  { shouldRender: async () => (await import('./lib/routing-render-checks')).shouldRenderRedirectPage(), component: LazyRedirectPage }
+  { shouldRender: async () => (await import('./lib/routing-render-checks.js')).shouldRenderRedirectsInterstitial(), component: LazyInterstitial },
```

Comment:
We could probably import this check function immediately since it's tiny but this is consistent at least.

Reply:
Sounds like a reasonable way to avoid an additional round trip.